cmd/systray/command/command.go

```diff
@@ -105,7 +105,7 @@ func MakeCommand() *cobra.Command {
 					defaultpaths.DogstatsDLogFile,
 					defaultpaths.StreamlogsLogFile,
 				)),
-				fx.Supply(optional.NewNoneOption[autodiscovery.Component]()),
+				noopAutodiscover.Module(),
```

Comment:
@DataDog/windows-agent Could you validate that the autodicovery is not used at all in the systray command? 

I replaced with a noop implementation as it was previously using a non option, but I don't what is the full set of feature that this command do 😄